### ISSUE TITLE
Grid Tables are not working

### ISSUE THREAD

**Issue by adius** [haskell/haddock#755]
I posted the example table from the documentation to every position I could think of (prologue, before module, after module, ...) but it never got never picked up.

`Produced by Haddock version 2.18.1`

Is there anything else / special I need to do?

**Comment by alexbiehl** [comment]
Currenly there is no haddock released with table support. Did you buils from master?

**Comment by adius** [comment]
No, I guess in that case the problem is the documentation. I just assumed that it was for the latest stable version.

You should set the default  to "stable" (http://docs.readthedocs.io/en/stable/features.html#versions)

**Comment by hvr** [comment]
@alexbiehl it may be worth mentioning the minimum version of haddock some features require, i.e. with which haddock version something got introduced. That way we can keep defaulting to the bleeding edge docs.
